### ISSUE TITLE
Seasonal Cookies

### ISSUE THREAD

**Issue by Iyonia** [StardewModders/mod-ideas#584]
**Tl;dr: A cookie that changes it's picture (maybe it's description?) for each season, matching with the Stardew Valley Cookie Lore™ established in Evelyn's dialogue.** 

Unnecessarily long original post:

I've seen a few mods that are capable of switching out assets when certain prerequisites are met. For someone who is comfortable implementing that sort of thing, I imagine this could be pretty simple, and kind of charming! 

The idea is just that - a seasonal default replacement for the cookies in the game. Evelyn mentions a few times in the game that the cookies are her mothers recipe, and that she likes to make them in all kinds of seasonal shapes! She mentions flower shapes in spring, palm tree shapes in summer, and pumpkins in fall. 

Some concept ideas!:

**Spring**: Three cookies, maybe daffodils, or a daffodil, a tulip, and a strawberry to mix things up. Or, for giggles, maybe have one of the cookies be shaped like a leek as an allusion to George's love for them. Or just pretty flower shapes!
**Summer**: Three cookies - a palm tree, a smiling sun, maybe a little boat or a seashell.
**Fall**: Pumpkin, bat, maybe a ghost or a cat
**Winter**: A star, maybe a tree, and the sign of the vessel - or maybe themed after some of the stuff seen during the three day festival (mermaids, candycanes, weird sealife) - I feel like the different images for a lot of things are pretty much a "season to taste" kind of situation.

I'd love to do the art for these, although I don't really have any experience making items like this, so I'd want to do a bit of looking into the dimensions/resolution I need. I'll post some concept pieces once the sun is up, and anyone is totally free to use them/ask for adjustments, or anything! Or don't use em, either way works for me.

**Comment by Jonqora** [comment]
This is *very* doable, and a great mod idea to tackle for a beginner just starting out!

You will want to create each cookie image as a 16x16 pixel sprite with a transparent background. Once you have your seasonal versions, drop by the #making-mods channel in the SDV discord and we can help you with structuring a Content Patcher pack to apply those sprites seasonally.

(If you want, you could even make the cookies change weekly, or have special versions around festival days, or delay each change a few days after the season starts... but that sort of thing adds a bit more complexity to the setup. Doing just 4 seasonal versions is a great and simple place to start.)

**Comment by methseyers** [comment]
If you are still wanting to make this idea into a mod, I'd be happy to work on it! Let me know and we can start the process! :)

**Comment by irocendar** [comment]
[(CP) Evelyn's Cookies](https://www.nexusmods.com/stardewvalley/mods/9174) does this now!
